modules/aruco/tutorials/aruco_detection/aruco_detection.markdown

```diff
@@ -15,7 +15,7 @@ The aruco module is based on the [ArUco library](http://www.uco.es/investiga/gru
 a popular library for detection of square fiducial markers developed by Rafael Muñoz and Sergio Garrido:
 
 > S. Garrido-Jurado, R. Muñoz-Salinas, F. J. Madrid-Cuevas, and M. J. Marín-Jiménez. 2014.
-> "Automatic generation and detection of highly reliable fiducial markers under occlusion".
+> “Automatic generation and detection of highly reliable fiducial markers under occlusion”.
```

Comment:
Please use ASCII quotes: `"` (instead of unicode symbols)

Reply:
Done. I found another set and changed those too.